BudgetHelper/src/test/resources/hibernate.properties

```diff
@@ -2,4 +2,7 @@ jdbc.driverClassName=org.postgresql.Driver
 jdbc.url=jdbc:postgresql://localhost:5432/budgethelper_test
 jdbc.username=postgres
 jdbc.password=root
+
 hibernate.dialect=org.hibernate.dialect.PostgreSQLDialect
+hibernate.show_sql=false
+hibernate.format_sql=true
```

Comment:
Please, add an epty line to the end of this file. Pay attention that git system set some special simbol in such cases.

Reply:
done